### ISSUE TITLE
[feature request] Proposed integration for Zidoo devices

### ISSUE THREAD

**Issue by albaintor** [unfoldedcircle/bug-reports#359]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I have created a python driver for Zidoo devices. It is working well on my setup (uses polling interval) and supports all the possible features of the media player entity and multiple instances
https://github.com/albaintor/integration-zidoo

**Comment by Tobbes1990** [comment]
At the moment, is it possible to add custom integrations?

**Comment by albaintor** [comment]
Yes I described the procedure here : https://github.com/albaintor/integration-sonyavr/wiki/How-to-create-an-external-driver-for-the-Remote-Two
It works pretty well

**Comment by zehnm** [comment]
A community integration covers this now
